pkg/nsx/util/license.go

```diff
@@ -0,0 +1,65 @@
+package util
+
+import (
+	"sync"
+)
+
+const (
+	FeatureContainer        = "CONTAINER"
+	FeatureDFW              = "DFW"
+	LicenseContainerNetwork = "CONTAINER_NETWORKING"
+	LicenseDFW              = "DFW"
+	LicenseContainer        = "CONTAINER"
+)
+
+var (
+	container_network   = false
+	container_security  = false
+	licenseMutex        sync.Mutex
+	licenseMap          = map[string]bool{FeatureContainer: container_network, FeatureDFW: container_security}
+	Feature_license_map = map[string][]string{FeatureContainer: {LicenseContainerNetwork,
+		LicenseContainer},
+		FeatureDFW: {LicenseDFW}}
+	Features_to_check = []string{FeatureContainer, FeatureDFW}
```

Comment:
could we just get the keys from `Feature_license_map`?

Reply:
done